### PR TITLE
fix regression

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

File: src/kernel-loader/index.ts [hunk 1]
```diff
@@ -252,8 +252,17 @@ async function initKernel() {
   })
 
   kernel.on('openUrl', ({ url }) => {
-    const newWindow = window.open(url, '_blank', 'noopener,noreferrer')
-    if (newWindow != null) newWindow.opener = null
+    try {
+      const u = new URL(url)
+      if (u.protocol === 'https:') {
+        const newWindow = window.open(u.toString(), '_blank', 'noopener,noreferrer')
+        if (newWindow != null) newWindow.opener = null
```
Comment (2fd) [inline src/kernel-loader/index.ts:259]: this line may fail if the opened URL doesn't have the same domain

Screen Shot 2022-08-25 at 22 41 23

**Review by 2fd — COMMENTED** [review]
@menduz just a little comment

File: src/kernel-loader/index.ts (hunk 1 above)
Comment (2fd) [reply src/kernel-loader/index.ts:259]: are we ok with this? Or do you prefer to change it?

Comment (menduz) [reply src/kernel-loader/index.ts:259]: I don't understand, it shouldn't fail. It has an if `if (newWindow != null) newWindow.opener = null`

Comment (2fd) [reply src/kernel-loader/index.ts:259]: should be `null` because of the `noopener` but if not and we try to overwrite `newWindow.opener`  it may fail

Comment (2fd) [reply src/kernel-loader/index.ts:259]: shouldn't we add a `try/catch` and close it if we couldn't secure the interactions with the new window?

Comment (2fd) [reply src/kernel-loader/index.ts:259]: Or remove the line entirely

**Review by 2fd — APPROVED** [review]

**Pull request merged**